util/util.go

```diff
@@ -472,11 +473,31 @@ func UnsharedRunrootPath(uid string) (string, error) {
 	if envRuntimeDir, haveRuntimeDir := os.LookupEnv("XDG_RUNTIME_DIR"); haveRuntimeDir {
 		return filepath.Join(envRuntimeDir, "run"), nil
 	}
+	var runtimeDir string
 	// If $XDG_RUNTIME_DIR is not defined, but we know our UID...
 	if uid != "" {
-		return filepath.Join("/var/run/user", uid, "run"), nil
+		tmpDir := filepath.Join("/var/run/user", uid, "run")
```

Comment:
Podman doesn't seem to have "run" on the end, is this correct or did I misread the podman code?

Reply:
I've left this part similar to the previous code in Buildah.  Should I drop the last "run"?

/cc @nalind

Reply:
thinking more of it, it seems correct the version without the "run" so I've pushed a new version now.  Thanks for catching this issue!